masui/src/main/java/com/ca/mas/ui/MASLoginActivity.java

```diff
@@ -285,7 +292,9 @@ public void onSuccess(MASUser result) {
             public void onError(Throwable e) {
                 progress.dismiss();
                 Toast.makeText(MASLoginActivity.this, "Login failed: " + e.getMessage(), Toast.LENGTH_SHORT).show();
-                MAS.cancelRequest(mRequestId);
+                Bundle data = new Bundle();
+                data.putString("LOGIN_STATUS", "FAILED");
```

Comment:
Is this LOGIN_STATUS retrieved elsewhere? Should we save the value as a constant?